test/integration/crossPartition.spec.ts

```diff
@@ -867,6 +724,103 @@ describe("Cross Partition", function() {
       }
     });
 
+    it("Validate simple LIMIT OFFSET", async function() {
```

Comment:
Can we add one test for an OFFSET LIMIT query with a logical partition key?

Reply:
There is a larger refactor that needs to happen to support this test. Will add a TODO for it.